book/src/dev/rfcs/0005-state-updates.md

```diff
@@ -606,7 +606,7 @@ We use the following rocksdb column families:
 | `height_by_hash`     | `block::Hash`         | `BE32(height)`                      |
 | `block_by_height`    | `BE32(height)`        | `Block`                             |
 | `tx_by_hash`         | `transaction::Hash`   | `(BE32(height) \|\| BE32(tx_index))`|
-| `utxo_by_outpoint`   | `OutPoint`            | `TransparentOutput`                 |
+| `utxo_by_outpoint`   | `OutPoint`            | `transparent::Output`               |
```

Comment:
I reverted this change and moved it to another state RFC PR to avoid conflicts.
```suggestion
| `utxo_by_outpoint`   | `OutPoint`            | `TransparentOutput`               |
```